my-app/src/app/courses-page/course-item/course-item-host-test.component.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { Course } from '../../models/course.interface';
+import { CourseItemComponent } from './course-item.component';
+
+@Component({
+  template: `
+    <app-course-item
+      [course]="course"
+      (edit)="onCourseEdit($event)"
+      (delete)="onCourseDelete($event)"
+    ></app-course-item>
+  `,
+})
+class CourseItemHostTestComponent {
+  course: Course = {
+    id: 1,
+    title: 'Video Course 1. Name tag',
+    creationDate: new Date('2020/10/10'),
+    duration: { hours: 1, minutes: 28 },
+    description: 'test description',
+  };
+
+  mockDate = {
+    day: this.course.creationDate.getDate(),
+    month: this.course.creationDate.getMonth() + 1,
+    year: this.course.creationDate.getFullYear().toString().substr(-2),
+  };
+
+  editedCourse: Course;
+  deletedCourse: Course;
+
+  onCourseEdit(course: Course): void {
+    this.editedCourse = course;
+  }
+
+  onCourseDelete(course: Course): void {
+    this.deletedCourse = course;
+  }
+}
+
+describe('CourseItemComponent', () => {
+  let testingHost: CourseItemHostTestComponent;
+  let fixture: ComponentFixture<CourseItemHostTestComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [CourseItemComponent, CourseItemHostTestComponent],
+    });
+    fixture = TestBed.createComponent(CourseItemHostTestComponent);
+    testingHost = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should show correct title, date & time', () => {
```

Comment:
Usually `it` should include one `expect` statement. It's fine here to have several but in real life, the test isn't gonna be that simple =)
Just a note, you can leave the code as is)